schema/artist/statuses.js

```diff
@@ -10,62 +10,61 @@ const ArtistStatusesType = new GraphQLObjectType({
   fields: {
     artworks: {
       type: GraphQLBoolean,
-      resolve: ({ id }) => {
-        return gravity(`artist/${id}/artworks`, {
-          published: true,
-          size: 1,
-        }).then(artworks => !!artworks.length);
-      },
+      resolve: ({ published_artworks_count }) => !!published_artworks_count,
     },
     shows: {
       type: GraphQLBoolean,
-      resolve: ({ id }) => {
-        return gravity('related/shows', {
-          artist_id: id,
-          displayable: true,
-          size: 1,
-        }).then(shows => !!shows.length);
-      },
+      resolve: ({ id }) => !!total(`related/shows`, {
+        artist_id: id,
+        displayable: true,
+        size: 0,
+      }),
+    },
+    cv: {
+      type: GraphQLBoolean,
+      resolve: ({ id }) => total(`related/shows`, {
+        artist_id: id,
+        displayable: true,
+        size: 0,
+      }).then(count => count > 15),
     },
     artists: {
       type: GraphQLBoolean,
-      resolve: ({ id }) => {
-        return gravity(`related/layer/main/artists`, {
-          exclude_artists_without_artworks: true,
-          artist: [id],
-        }).then(artists => !!artists.length);
-      },
+      resolve: ({ id }) => !!total(`related/layer/main/artists`, {
+        exclude_artists_without_artworks: true,
+        artist: [id],
+        size: 0,
+      }),
     },
     contemporary: {
       type: GraphQLBoolean,
-      resolve: ({ id }) => {
-        return gravity(`related/layer/contemporary/artists`, {
-          exclude_artists_without_artworks: true,
-          artist: [id],
-        }).then(contemporary => !!contemporary.length);
-      },
+      resolve: ({ id }) => !!total(`related/layer/contemporary/artists`, {
+        exclude_artists_without_artworks: true,
+        artist: [id],
+        size: 0,
+      }),
     },
     articles: {
       type: GraphQLBoolean,
-      resolve: ({ _id }) => {
-        return positron('articles', {
-          published: true,
-          artist_id: _id,
-        }).then(articles => !!articles.count);
-      },
+      resolve: ({ _id }) => positron('articles', {
+        artist_id: _id,
+        published: true,
+        limit: 0,
+      }).then(({ count }) => !!count),
     },
     auction_lots: {
       type: GraphQLBoolean,
-      resolve: ({ display_auction_link }) => display_auction_link,
+      resolve: ({ display_auction_link, auction_lots_count }) => {
+        return display_auction_link && !!auction_lots_count;
```

Comment:
We don't even return `auction_lots_count` in the JSON anymore. It's just `display_auction_link`.